inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java

```diff
@@ -588,8 +588,8 @@ private ProxyConfigEntry getLocalProxyListFromFile(String filePath) throws Excep
         Map<String, Integer> streamIdMap = getStreamIdMap(localProxyAddrJson);
         proxyEntry.setGroupIdNumAndStreamIdNumMap(groupIdNum, streamIdMap);
         proxyEntry.setLoad(load);
-        if (localProxyAddrJson.has("cluster_id")) {
-            proxyEntry.setClusterId(localProxyAddrJson.get("cluster_id").getAsString());
+        if (localProxyAddrJson.has("clusterId")) {
```

Comment:
Please modify the demo such as TcpClientExample、HttpClientExample and the instructions in the documentation at the same time.

Reply:
demo package is org.apache.inlong.sdk.dataproxy.example

Reply:
The github project where the documentation is located is: https://github.com/apache/incubator-inlong-website